tests/TestCase/Shell/BakeShellTest.php

```diff
@@ -87,7 +87,7 @@ public function testAllWithModelName()
             ->with('Comments');
 
         $this->Shell->connection = '';
-        $this->Shell->params = [];
+        $this->Shell->params = ['prefix' => 'account'];
```

Comment:
@andtxr The PR looks good but this doesn't look to really affect the actual test ? Am I missing something ?